src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyPlainTextCommentFilter.java

```diff
@@ -0,0 +1,663 @@
+     * @return {@link Optional} of {@link Suppression}.
+     */
+    private Optional<Suppression> getSuppression(FileText fileText, int lineNo) {
+        final String line = fileText.get(lineNo);
+        final Matcher commentMatcher = commentFormat.matcher(line);
+
+        Suppression suppression = null;
+        if (commentMatcher.find()) {
+            suppression = new Suppression(commentMatcher.group(0), lineNo + 1, this);
+        }
+
+        return Optional.ofNullable(suppression);
+    }
+
+    /**
+     * Finds the nearest {@link Suppression} instance which can suppress
+     * the given {@link AuditEvent}. The nearest suppression is the suppression which scope
+     * is before the line and column of the event.
+     *
+     * @param suppressions collection of {@link Suppression} instances.
+     * @param event {@link AuditEvent} instance.
+     * @return {@link Suppression} instance.
+     */
+    private static Suppression getNearestSuppression(Collection<Suppression> suppressions,
+                                                    AuditEvent event) {
+        return suppressions
+            .stream()
+            .filter(suppression -> suppression.isMatch(event))
+            .findFirst()
+            .orElse(null);
+    }
+
+    /** The class which represents the suppression. */
+    private static final class Suppression {
+
+        /** The first line where warnings may be suppressed. */
+        private final int firstLine;
+
+        /** The last line where warnings may be suppressed. */
+        private final int lastLine;
+
+        /** The regexp which is used to match the event source.*/
+        private final Pattern eventSourceRegexp;
+
+        /** The regexp which is used to match the event message.*/
+        private final Pattern eventMessageRegexp;
+
+        /** The regexp which is used to match the event ID.*/
+        private final Pattern eventIdRegexp;
+
+        /**
+         * Constructs new {@code Suppression} instance.
+         *
+         * @param text suppression text.
+         * @param lineNo suppression line number.
+         * @param filter the {@code SuppressWithNearbyPlainTextCommentFilter} with the context.
+         * @throws IllegalArgumentException if there is an error in the filter regex syntax.
+         */
+        /* package */ Suppression(
+                String text,
+                int lineNo,
+                SuppressWithNearbyPlainTextCommentFilter filter
+        ) {
+            final Pattern commentFormat = filter.commentFormat;
+            final String influenceFormat = filter.influenceFormat;
+            String format = "";
+            try {
+                format = CommonUtil.fillTemplateWithStringsByRegexp(
+                        filter.checkFormat, text, commentFormat);
+                eventSourceRegexp = Pattern.compile(format);
+                if (filter.messageFormat == null) {
+                    eventMessageRegexp = null;
```

Comment:
```
Source File: "SuppressWithNearbyPlainTextCommentFilter.java"
Class: "com.puppycrawl.tools.checkstyle.filters.SuppressWithNearbyPlainTextCommentFilter$Suppression"
Method: "&lt;init&gt;"
Line Contents: "eventIdRegexp = null;"
Mutator: "org.pitest.mutationtest.engine.gregor.mutators.experimental.MemberVariableMutator"
Description: "Removed assignment to member variable eventIdRegexp"
Line Number: 527
```

From [EXPERIMENTAL_MEMBER_VARIABLE](https://pitest.org/quickstart/mutators/#EXPERIMENTAL_MEMBER_VARIABLE) docs:
>Please Note: This mutator is likely to create equivalent mutations if a member variable is explicitly initialized with the Java default value for the specific type of the member variable

I think this mutation can be safely suppressed?